CODEOWNERS

```diff
@@ -1,7 +1,7 @@
-*  @openvinotoolkit/nncf_pytorch-maintainers
+*  @openvinotoolkit/nncf-maintainers
 
 CODEOWNERS  @openvinotoolkit/openvino-admins
 
 # Control 3d party dependencies
-requirements.txt  @openvino-configuration-mgmt
-**/setup.py  @openvino-configuration-mgmt
+requirements.txt  @openvinotoolkit/openvino-configuration-mgmt
+**/setup.py  @openvinotoolkit/openvino-configuration-mgmt
```

Comment:
Remove this altogether. No one in these teams seemed to care before and I don't see why they would be caring now; at least there is no defined process for it.

Reply:
I am ok

Reply:
I'm ok.